.gitignore

```diff
@@ -1 +1,9 @@
+# Rust ignore
 /target
+**/*.rs.bk
+Cargo.lock
```

Comment:
we have `Cargo.lock` versioned since we are shipping a binary, but also I see more libraries versioning `Cargo.lock` as semver sometimes fails,  and also for caching in the CI.  
In our case we require it to be updated also, see [here](https://github.com/polyphene/kythera/pull/11) Philippe.

Reply:
thanks for the detailed information. updating 👍